public/fonts/material_icons/material_icons.css

```diff
@@ -17,6 +17,11 @@
     url(/fonts/material_icons/MaterialIconsOutlined-Regular.otf) format("opentype");
 }
 
+.material-icon-loaded .material-icons,
+.material-icon-loaded .material-icons-outlined {
+  opacity: 1;
```

Comment:
I'm not sure if `opacity` or `visibility` is better for accessibility, but either way, we should probably add a `width: 1rem` so that there is less UI jumping.

Reply:
oh interesting, I assumed opacity would not affect layout